README.md

```diff
@@ -221,13 +221,13 @@ Notes from all past meetings are available [here](https://docs.google.com/docume
 Here is the list of organizations that are using Amundsen today. If your organization uses Amundsen, please file a PR and update this list.
 
 Currently **officially** using Amundsen:
-
 1. [Asana](https://asana.com/)
 1. [Bagelcode](https://site.bagelcode.com/)
 1. [Bang & Olufsen](https://www.bang-olufsen.com/en)
 1. [Brex](https://www.brex.com/)
 1. [Cameo](https://www.cameo.com)
 1. [Cimpress Technology](https://cimpress.com)
+1. [Chan Zuckerberg Initiative](https://chanzuckerberg.com)
```

Comment:
could you make it sorted alphabetically? thanks , great to see your team onboard to Amundsen!

Reply:
lol i don't know how to alphabetize 🤦‍♀️